rules/linux/process_creation/proc_creation_lnx_gtfobin_ssh_shell.yml

```diff
@@ -0,0 +1,36 @@
+title: SSH GTFOBin Abuse - Linux
+id: 8737b7f6-8df3-4bb7-b1da-06019b99b687
+status: experimental
+description: Detects usage of SSH and variants as a GTFOBin Linux binary abuse to break out from restricted environments by spawning an interactive system shell. 
+references:
+    - https://gtfobins.github.io/gtfobins/ssh/
+    - https://www.elastic.co/guide/en/security/current/linux-restricted-shell-breakout-via-linux-binary-s.html
+author: Li Ling, Andy Parkidomo, Robert Rakowski, Blake Hartstein (Bloomberg L.P.)
+date: 2024/08/05
+tags:
+    - attack.execution
+    - attack.t1059
+logsource:
+    category: process_creation
+    product: linux
+detection:
+    selection_img:
+        Image|endswith: '/ssh'
+        CommandLine|contains:
+            - 'ProxyCommand=;'
```

Comment:
Need a log to check but I think ";" will split the commandline in 2 logs.

Reply:
We checked the logs, we can see the original log with commandline contains "ProxyCommand=;", and as well as child processes, see below example logs.

This rule is aiming to detect three SSH variants (https://gtfobins.github.io/gtfobins/ssh/#shell):

SSH Variant 1:
process_image :  /usr/bin/ssh                                                                         
process_cmdline  : ssh localhost /bin/bash --noprofile --norc   
SSH Variant 2:
process_image : /usr/bin/ssh                                                                         
process_cmdline  : ssh -o "proxycommand=;sh 0<&2 1>&2" x  
SSH Variant 3:
process_image  : /usr/bin/ssh                                                                         
process_cmdline  : ssh -o permitlocalcommand=yes -o localcommand=/bin/sh xxxx.com